app/controllers/api/v1/stops_controller.rb

```diff
@@ -1,7 +1,12 @@
 class Api::V1::StopsController < ApplicationController
 
   def show
-    render json: Stop.find(params[:id])
+
+    @stop = Stop.find(params[:id])
+    @review = @stop.reviews
+    response = { :stop => @stop, :reviews => @review }
+    render :json => response
+
```

Comment:
please update syntax `=>`, otherwise known as hash rockets, is deprecated. `=>` was the old way of setting key value pairs, but you should do `{ stop: @stop, reviews: @review }`